src/components/ImageMap.tsx

```diff
@@ -27,40 +20,60 @@ function ImageMap() {
 
   return (
     <Container>
-      <Image src={data.imageUrl} />
-      {products?.map((el, idx) => {
+      <Image src={productData.imageUrl} />
+      {products?.map((products: IProductList) => {
```

Comment:
products 의 타입이 IProductList[] 이기 때문에

map 을 돌릴 때 개별적으로 나오는 value 는 []이 아닌 IProductList 타입이 됩니다.

그래서 굳이 작성하지 않아도 해당 타입이 적용됩니다.

추가로,
IProductList 는 IProduct[] 의 의미하고 더 가깝기 때문에
IProductList 이름은 IProduct 로 변경하는 것이 좋을 것 같습니다.

그리고 다음과 같이
```suggestion
      {products?.map((product) => {
```
배열 안에 도는 요소의 이름은 단수형으로 작성하시면 되겠습니다.

Reply:
넵 수정해서 다시 적용해보겠습니다 확인 감사합니다! :)